src/js/disability-benefits/containers/YourClaimsPage.jsx

```diff
@@ -173,14 +174,19 @@ class YourClaimsPage extends React.Component {
 
     return (
       <div className="your-claims">
+        <Breadcrumbs/>
```

Comment:
It looks like the class nesting is slightly different here. Does the `your-claims` class serve a styling purpose? Does the specificity override other styling?

Reply:
Yeah, `your-claims` overwrites some styles in its children (a lot of the css for this app is a tangled mess). I'm going to create a tech debt ticket for that.